[WIP] fix mistakes (#146)
AdguardTeam/KnowledgeBaseVPN: docs/adguard-vpn-for-ios/solving-problems/automation.md

sidebar_label: How to set up AdGuard VPN automation
---

> There are no app exclusions in AdGuard VPN for iOS. Our desktop apps have the *Split tunneling* module and the app for Android has *Apps settings* — these settings allow you to decide for which apps AdGuard VPN should be enabled. Yet, there is an alternative way to automate AdGuard VPN for apps on iPhones and iPads.

Reviewer: > Our desktop apps have the *Split tunneling* module and the app for Android has *Apps settings* — these settings allow you to decide for which apps AdGuard VPN should be enabled.

а ты это не исправляла? это все устаревшая инфа

sidebar_label: How to set up AdGuard VPN automation
---

> AdGuard VPN has an *Exclusions* section and two operation modes — *General* and *Selective*. In *General mode* AdGuard VPN works everywhere except sites added to exclusions. Conversely, in *Selective mode*, VPN doesn't work anywhere except sites listed in the exclusions list. Note, that for each mode you must create a separate list.

Reviewer: выпилила бы нафиг это вступление, оно не в тему

As you may notice, only websites can be added to the *Exclusions* section. To adjust AdGuard VPN for apps you need to use another feature. Our desktop apps have the *Split tunneling* module and the app for Android has *Apps settings* — these settings allow you to decide for which apps AdGuard VPN should be enabled.

> But, as it often happens, due to a number of technical nuances, it is impossible to implement such a useful function for iOS, at least for now. Therefore, we offer you an alternative way to automate AdGuard VPN for apps on iPhones and iPads.

Reviewer: технические нюансы можно унести в предыдущее предложение или выпилить. оставить только инфу «вот альтернативный способ»

3. In the next window, make sure that the *Is Opened* option is selected, and then tap *Choose* to choose the app.

![Instruction. Part 2](https://cdn.adguardvpn.com/public/Adguard/Blog/VPNauto/vpn_on2_en.jpg)

> 4. Start entering the name of the application, in our case it's Twitter, and select it. Then tap *Done* in the upper right corner of the screen. Then tap *Next* in the upper right corner of the screen. And in the opened window tap *Add Action*.

Reviewer: ```suggestion
4. Start entering the name of the app (in our case it's Twitter) and select it. Tap *Done*, then tap *Next* in the upper right corner of the screen. In the opened window, tap *Add Action*.
```

![Instruction. Part 3](https://cdn.adguardvpn.com/public/Adguard/Blog/VPNauto/vpn_on3_en.jpg)
5. Start entering “AdGuard VPN” and select the AdGuard VPN app. In the new window tap *Set a VPN connection*.

![Instruction. Part 4](https://cdn.adguardvpn.com/public/Adguard/Blog/VPNauto/vpn_on4_en.jpg)
6. Make sure the variables say *Turn* VPN connection *On* and tap *Next*.
7. In the next window, move the slider next to the *Ask before running* option to the inactive position. Confirm your choice, then tap *Done*.

Now you have a new scenario: AdGuard VPN will be automatically enabled when you start the Twitter app. Now you need to create another command that will make AdGuard VPN automatically turn off when you close the app.

## Configuring AdGuard VPN automatic shutdown

![Instruction. Part 1](https://cdn.adguardvpn.com/public/Adguard/Blog/VPNauto/vpn_off1_en.jpg)

> 1. In the same *Shortcuts* app start creating a new automation: tap *+* in the upper right corner of the screen and then on the *Create Personal Automation* button. In the opened window choose *App*.

Reviewer: ```suggestion
1. In the same *Shortcuts* app, start creating a new automation: tap *+* in the upper right corner of the screen, then select *Create Personal Automation*. In the opened window, select *App*.
```

кстати на 17 иосе после + нет кнопки Create Personal Automation
